coreneuron/CMakeLists.txt

```diff
@@ -71,7 +62,7 @@ macro(mod2c_target name input)
     list(APPEND MOD2C_${name}_MODS "${mod2c_modname_}")
 
     add_custom_command(OUTPUT "${mod2c_output_}"
-      DEPENDS ${input} "${mod2c_install_prefix}/mod2c_core"
+      DEPENDS mod2c
```

Comment:
wondering if we need ${MOD2C} exe here instead of just mod2c. (but this is already working).

Why ${input} was removed? because we still depend on input mod file